src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -374,7 +374,9 @@ public static bool Exists([NotNull]string name, [CanBeNull]string directory)
         [ExcludeFromCodeCoverage]
 		public static void SetLogLevel(LogDomain domains, LogLevel level)
 		{
-		    WriteLog.To.Database.W(Tag, "Database.SetLogLevel is deprecated, use Database.Log.Console instead.");
+            Log.Console.Level = level;
```

Comment:
did you forget the indentation?

Reply:
The IDE automatically handles all the indentation.... They look fine in my codebase... but not sure what happens here...

Reply:
Check tabs vs spaces....this is a very common problem.